### PR TITLE
[18.05] Add io_details and link_details in api tools doc

### PR DESCRIPTION
**Pull request by ValentinChCloud**
Fix missing options in tools api show function in documentations. #6218

Thx @mvdbeek

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/api/tools.py [hunk 1]
```diff
@@ -86,8 +86,15 @@ def index(self, trans, **kwds):
...
-        GET /api/tools/{tool_id}
-        Returns tool information, including parameters and inputs.
+        GET /api/tools/{tool_id} 
+
+        Returns tool information
+
+            parameters:
+
+                io_details   - if true, parameters and inputs are returned
+
```
Comment (mvdbeek) [inline lib/galaxy/webapps/galaxy/api/tools.py]: Can you skip this newline here ?

File: lib/galaxy/webapps/galaxy/api/tools.py [hunk 2]
```diff
@@ -86,8 +86,15 @@ def index(self, trans, **kwds):
     @expose_api_anonymous_and_sessionless
     def show(self, trans, id, **kwd):
         """
-        GET /api/tools/{tool_id}
-        Returns tool information, including parameters and inputs.
+        GET /api/tools/{tool_id}
```
Comment (mvdbeek) [inline lib/galaxy/webapps/galaxy/api/tools.py]: There's a whitespace after `/api/tools/{tool_id}`, you need to remove this for the linting to not raise an error.
Also this should be `/api/tools/{tool_id}/show`

**Review by mvdbeek — CHANGES_REQUESTED** [review]
Looks better already, just 2 minor changes needed

File: lib/galaxy/webapps/galaxy/api/tools.py [hunk 3]
```diff
@@ -86,8 +86,14 @@ def index(self, trans, **kwds):
     @expose_api_anonymous_and_sessionless
     def show(self, trans, id, **kwd):
         """
-        GET /api/tools/{tool_id}
-        Returns tool information, including parameters and inputs.
+        GET /api/tools/{tool_id}/show
```
Comment (mvdbeek) [inline lib/galaxy/webapps/galaxy/api/tools.py]: Sorry, I'm stupid, it was actually correct before

Comment (ValentinChCloud) [reply lib/galaxy/webapps/galaxy/api/tools.py]: Okay ^^", I do that

**Review by mvdbeek — APPROVED** [review]

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**